resources/hiveclients/java/hiveserver1-client/README.md

```diff
@@ -0,0 +1,26 @@
+#hiveserver1-client
```

Comment:
I will put HiveServer1 client since the official name of the server is "HiveServer1"